docs/references/compose-spec.md

```diff
@@ -623,7 +626,7 @@ volumes:
 
 !!! Warning  
 
-    If more than one service mounts a shared volume, only one service can have read-write access to it. All other services must be designated read-only, e.g. using the sh~ort syntax ` - shared_db:/var/lib/mysql:ro` or long syntax `read-only: true`.
+    The options 'source' and 'target' are required for a long syntax. Therefore anonymous volume can't be define for a long syntax.
```

Comment:
The 'source' and 'target' options are required for the long syntax. There for you can't use long syntax with anonymous volumes.